### ISSUE TITLE
go get -u github.com/jinzhu/gorm not working

### ISSUE THREAD

**Issue by etsrepo** [jinzhu/gorm#2398]
### What version of Go are you using (`go version`)?
go version go1.12.4 darwin/amd64

### Log of `go get -u github.com/jinzhu/gorm`

```
go: finding github.com/jinzhu/inflection latest
go: finding github.com/denisenkom/go-mssqldb latest
go: finding github.com/jstemmer/go-junit-report latest
go: finding gopkg.in/check.v1 latest
go: finding golang.org/x/sync latest
go: finding golang.org/x/sys latest
go: finding golang.org/x/crypto latest
go: finding golang.org/x/exp latest
go: finding google.golang.org/api v0.0.0-20181030000543-1d582fd0359e
go: finding honnef.co/go/tools latest
go: finding golang.org/x/lint latest
go: finding golang.org/x/net latest
go: finding golang.org/x/build latest
go: finding go4.org latest
go: finding golang.org/x/tools latest
go: finding golang.org/x/time latest
go: finding github.com/bradfitz/go-smtpd latest
go: finding grpc.go4.org latest
go: finding github.com/golang/glog latest
go: finding golang.org/x/oauth2 latest
go: finding github.com/flynn/go-shlex latest
go: finding github.com/BurntSushi/xgb latest
go: finding github.com/coreos/go-systemd latest
go: finding github.com/anmitsu/go-shlex latest
go: finding golang.org/x/image latest
go: finding github.com/beorn7/perks latest
go: finding github.com/xdg/scram latest
go: finding github.com/prometheus/client_model latest
go: finding github.com/jellevandenhooff/dkim latest
go: finding github.com/prometheus/procfs latest
go: finding google.golang.org/genproto latest
go: finding github.com/eapache/go-xerial-snappy latest
go: finding google.golang.org/api v0.0.0-20181220000619-583d854617af
go: finding github.com/rcrowley/go-metrics latest
go: finding github.com/tarm/serial latest
go: finding golang.org/x/perf latest
go: finding golang.org/x/mobile latest
go: finding github.com/gregjones/httpcache latest
go: finding github.com/kr/logfmt latest
go: finding github.com/alecthomas/template latest
go: finding github.com/aclements/go-gg latest
go: finding github.com/gonum/blas latest
go: finding github.com/gonum/matrix latest
go: finding github.com/kr/pty v1.1.1
go: finding github.com/GoogleCloudPlatform/cloudsql-proxy latest
go: finding github.com/gonum/lapack latest
go: finding github.com/mwitkow/go-conntrack latest
go: finding gopkg.in/tomb.v1 latest
go: finding github.com/alecthomas/units latest
go: finding cloud.google.com/go v0.0.0-20170206221025-ce650573d812
go: finding github.com/gonum/internal latest
go: finding github.com/google/pprof latest
go: finding github.com/golang/lint latest
go: github.com/golang/lint@v0.0.0-20190409202823-959b441ac422: parsing go.mod: unexpected module path "golang.org/x/lint"
go: finding github.com/gonum/floats latest
go: finding github.com/aclements/go-moremath latest
go: cloud.google.com/go@v0.0.0-20170206221025-ce650573d812: git -c protocol.version=0 fetch --unshallow -f https://code.googlesource.com/gocloud refs/heads/*:refs/heads/* refs/tags/*:refs/tags/* in /Users/cp/go/pkg/mod/cache/vcs/b0e27935eb83c1d7843713bafab507e95768b550f0552cb42d9f41e5fd9c8375: exit status 128:
	fatal: bad object a5913b3f7deecba45e98ff33cefbac4fd204ddd7
	error: remote did not send all necessary objects
go: google.golang.org/api@v0.0.0-20181030000543-1d582fd0359e: git -c protocol.version=0 fetch --unshallow -f https://code.googlesource.com/google-api-go-client refs/heads/*:refs/heads/* refs/tags/*:refs/tags/* in /Users/cp/go/pkg/mod/cache/vcs/9e62a95b0409d58bc0130bae299bdffbc7b7e74f3abe1ecf897474cc474b8bc0: exit status 128:
	fatal: bad object 19e022d8cf43ce81f046bae8cc18c5397cc7732f
	error: remote did not send all necessary objects
go: google.golang.org/api@v0.0.0-20181220000619-583d854617af: unknown revision 583d854617af
go get: error loading module requirements
```

**Comment by kkosmrli** [comment]
Can confirm this issue. I just ran into the same error. Tried on Linux and OSX with go 1.11 and 1.12.

**Comment by theocodes** [comment]
same

`
go: finding github.com/golang/lint v0.0.0-20180702182130-06c8688daad7
go: finding google.golang.org/api v0.0.0-20181030000543-1d582fd0359e
go: finding google.golang.org/api v0.0.0-20181220000619-583d854617af
go: finding google.golang.org/api v0.0.0-20180910000450-7ca32eb868bf
go: finding golang.org/x/lint v0.0.0-20180702182130-06c8688daad7
go: google.golang.org/api@v0.0.0-20181030000543-1d582fd0359e: git -c protocol.version=0 fetch --unshallow -f https://code.googlesource.com/google-api-go-client refs/heads/*:refs/heads/* refs/tags/*:refs/tags/* in /home/theocodes/Code/go/pkg/mod/cache/vcs/9e62a95b0409d58bc0130bae299bdffbc7b7e74f3abe1ecf897474cc474b8bc0: exit status 128:
	fatal: bad object 19e022d8cf43ce81f046bae8cc18c5397cc7732f
	error: remote did not send all necessary objects
go: google.golang.org/api@v0.0.0-20181220000619-583d854617af: unknown revision 583d854617af
go: google.golang.org/api@v0.0.0-20180910000450-7ca32eb868bf: unknown revision 7ca32eb868bf
go: error loading module requirements
`

Seems related to google.golang.org/api?

**Comment by kkosmrli** [comment]
When using `go get github.com/jinzhu/gorm@v1.9.2` the problem does not occur.

**Comment by mattdamon108** [comment]
I have same issue here.

**Comment by Siceberg** [comment]
https://github.com/golang/go/wiki/Resolving-Problems-From-Modified-Module-Path

**Comment by dahamr** [comment]
We have the same issue. I'm wondering if there's a plan for updating this? I can submit a PR from my fork if you like. I simply updated go-mssqldb to latest and can't reproduce anymore.

**Comment by emirb** [comment]
Thanks, fixed in #2428 and tagged as [v1.9.5](https://github.com/jinzhu/gorm/releases/tag/v1.9.5).
